### DISCUSSION TITLE
Generate JSON Schema

### DISCUSSION THREAD

**Discussion by linonetwo** [sinclairzx81/typebox#440] [Ideas]
So it can be used by [rjsf](https://github.com/rjsf-team/react-jsonschema-form)

**Comment by knight10-ux** [comment]
Not sure if I'm blind, but I couldn't find it in the docs either. I am looking now for some method which can dump this internal representation of a type as a json schema.

I need it also to store types in database as json.

PS. lol

```typescript
import { Type } from '@sinclair/typebox';

const tObject = Type.Object(
  {
    x: Type.Number(),
  },
  {
    $schema: 'http://example.com/json-schema.json',
    description: 'Example'
  },
);

console.log(JSON.stringify(tObject, null, 2));
```
